lib/python/rosie/vc.py

```diff
@@ -316,6 +352,10 @@ def generate_info_config(self, from_id=None, prefix=None):
                 from_project = from_config.get(["project"]).value
             if from_config.get(["title"]) is not None:
                 from_title = from_config.get(["title"]).value
+            if from_config.get(["description"]) is not None:
+                from_description = from_config.get(["description"]).value
+            if from_config.get(["sub-project"]) is not None:
```

Comment:
If I set a new field in my suite - e.g. `member_number` - does that get copied? I would expect that all properties get copied over unless they have a `copy-mode` set.